server/src/test/java/org/opensearch/env/NodeEnvironmentTests.java

```diff
@@ -566,10 +566,10 @@ public void testEnsureNoShardDataOrIndexMetadata() throws IOException {
         verifyFailsOnMetadata(noDataNoMasterSettings, indexPath);
 
         // build settings using same path.data as original but without master role
-        Settings noMasterSettings = nonMasterNode(settings);
+        Settings noClusterManagerSettings = nonMasterNode(settings);
```

Comment:
`nonMasterNode ` -> `nonClusterManagerNode `

Reply:
`nonMasterNode()` is a public method, so I plan to change it in the next phase when renaming all the public Java API, and targeted for 3.0 version.